### ISSUE TITLE
How to get stable and reproducible mAP in different training?

### ISSUE THREAD

**Issue by liyiwei979621500** [open-mmlab/mmdetection#3496]
I trained retinanet on my customed dataset 3 times using the same config file and --seed 0.

![image](https://user-images.githubusercontent.com/16919236/89439942-84f8c080-d77d-11ea-9379-fb9fb48366ef.png)

However
The mAP is fluctuating in single training phase.
The final mAP is quiet different in 3 training.

Can someone provide some suggestions making the mAP more stable and reproducible?

**Comment by ZwwWayne** [comment]
This might because the dataset is too small. In this case, we do not have a very concrete solution. You can choose to finetune the model with a pre-trained detector, and maybe further freeze some layers.

**Comment by ZwwWayne** [comment]
In a large dataset with seed, e.g., COCO, the fluctuation is very little (less than 0.3 mAP). So this is not the problem of the codebase itself.
